README.md

```diff
@@ -51,6 +45,36 @@ This will also install these package for the following tasks:
 
 If you choose not to install all packages now you will be prompted to do so whenever one is required.
 
+## Setting up
+
+Before loading **{laminr}** for the first time you should:
+
+1. Set up a Python environment
+
+```r
+laminr::install_lamindb()
+```
+
+2. Log in
+
+```r
+laminr::lamin_login(api_key = "your_api_key")
+```
+
+OR
+
+```r
+laminr::lamin_login(user = "user_handle")
```

Comment:
This is not going to work, so, I'd remove it. It makes things look complicated.

Reply:
I meanwhile understand that you mention it for switching users that had already been authenticated before. That works but is a total edge case that shouldn't be on the initial tutorial. So, I'm still voting for removing it here.

Reply:
As in, this exact code won't work (and just this needs updating) or switch users with a user name will never work (and I need to update the function)?

Reply:
The code will work and it's well explained in the `setup` guide. But it's an edge case that shouldn't show up in the first guide a user encounters who just wants to get going fast without reading through the entire `setup` instruction.